pandas/core/groupby/groupby.py

```diff
@@ -2637,7 +2636,13 @@ def describe(self, **kwargs):
             )
             if self.axis == 1:
                 return result.T
-            return result.unstack()
+
+            if self._selected_obj.ndim != 1 or self.as_index:
```

Comment:
Nit: `self._selected_obj.ndim` only has to be checked once?

```
if self._selected_obj.ndim != 1:
     if self.as_index:
          result = 
     else:
        ....
```

Would also be good to put the GH reference

Reply:
Good call - I've simplified. I think this also makes the intention more clear.